modules/gitops-release-notes-1-18-2.adoc

```diff
@@ -0,0 +1,82 @@
+
+* Grant access to the `argoproj.io/Application` resource
+* Are labeled with `app.kubernetes.io/part-of=argocd`
+* Bind to a service account in a different namespace, resulting in cross-namespace access
+
+Review the script output to ensure that cross-namespace access is limited to the intended namespaces only.
+
+For more information about the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/audit-operator-roles.sh[Audit Operator script].
+
+For more information about the steps to execute the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/README.md[Audit Operator script readme].
+====
+
+[id="errata-updates-1-18-2_{context}"]
+== Errata updates
+
+[id="RHSA-2025:23203-gitops-1-18-2-security-update-advisory_{context}"]
+=== RHSA-2025:23203 - {gitops-title} 1.18.2 security update advisory
+
+Issued: 2025-12-15
+
+The list of security fixes included in this release is available in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2025:23203[RHSA-2025:23203]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-18-2_{context}"]
+== Fixed issues
+
+Progressive Sync status and UI stability improvements::
+With this update, multiple issues are resolved where applications generated by `ApplicationSets` incorrectly displayed the Progressive Sync status as `Unknown` in the UI. The Progressive Sync state is now displayed correctly, and when the feature is disabled, the UI hides the field entirely. This update also fixes UI crashes that occur when Progressive Sync was enabled but the `applicationStatus` field is not available in the `ApplicationSet` status.
++
+link:https://issues.redhat.com/browse/GITOPS-7798[GITOPS-7798]
+
+git-lfs dependency update::
+* With this update, the `git-lfs` dependency in the Argo CD container image is updated to `v3.7.1`.
++
+link:https://issues.redhat.com/browse/GITOPS-8142[GITOPS-8142]
+
+Redis version upgraded to v7.2.11::
+* With this update, the Redis version in the {gitops-title} is upgraded to `v7.2.11`.
++
+link:https://issues.redhat.com/browse/GITOPS-8019[GITOPS-8019]
```

Comment:
This looks like enhancements rather than fixed issues. Consider moving the above two issues to a separate section like:

```
[id="enhancements-1-18-2_{context}"]
== Enhancements
```